googletest/src/gtest-port.cc

```diff
@@ -277,12 +279,12 @@ size_t GetThreadCount() {
 
 #endif  // GTEST_OS_LINUX
 
-#if GTEST_IS_THREADSAFE && GTEST_OS_WINDOWS
-
 void SleepMilliseconds(int n) {
-  ::Sleep(static_cast<DWORD>(n));
+  std::this_thread::sleep_for(std::chrono::nanoseconds(n));
```

Comment:
This sleeps for nanoseconds, not milliseconds.

Reply:
Thanks :)

Reply:
You're welcome! :)